### ISSUE TITLE
E492: Not an editor command: MatlabCliOpenInMatlabEditor

### ISSUE THREAD

**Issue by sadid** [daeyun/vim-matlab#22]
I've installed vim-matlab for neovim and I'm able to launch the matlab server and issue command. But I can't open this editor in my matlab (currently running) and `matlab` is already in the `$PATH`. 

here is the error:

```
E492: Not an editor command: MatlabCliOpenInMatlabEditor
```

Besides this (might be related). I don't have :MatlabCliOpenInMatlabEditor in my command mode. I just have these:
```vimscript
:MatlabInsertModeCreateCell  
:MatlabLaunchServer          
:MatlabNormalModeCreateCell  
:MatlabVisualModeCreateCell
```

I launch ` :MatlabCliOpenInMatlabEdito` with `,e` and get the above error. This happens for others as well such as `:MatlabCliRunSelection`.

**Comment by daeyun** [comment]
Can it be related to #20?

**Comment by sadid** [comment]
Yes, I can confirm the problem has been resolved by `:UpdateRemotePlugins` command. 

p.s. I'm a vim user and I've installed neovim just for this plugin. Thanks for the great work. 
BTW, it might be good to notice vim users in the README file to issue this command after installation.

**Comment by tdy** [comment]
It was in there already, but maybe #23 will emphasize that step a bit more.

**Comment by sadid** [comment]
@tdy, It has been added recently and now it's perfect. It's now on both FAQ and installation.
